src/GridFieldOrderableRows.php

```diff
@@ -360,7 +361,18 @@ public function getManipulatedData(GridField $grid, SS_List $list)
                 $sortterm .= $this->getSortTable($list).'.'.$this->getSortField();
             } else {
                 $sortterm .= '"'.$this->getSortTable($list).'"."'.$this->getSortField().'"';
+
+                if ($list instanceof DataList) {
+                    $classname = $list->dataClass();
+                    if (Config::inst()->get($classname, 'default_sort')) {
+                        // Append the default sort to the end of the sort string
+                        // This may result in redundancy... but it seems to work
+                        $sortterm .= $sortterm ? ', ' : '';
+                        $sortterm .= Config::inst()->get($classname, 'default_sort');
```

Comment:
```suggestion
                    if ($defaultSort = Config::inst()->get($classname, 'default_sort')) {
                        // Append the default sort to the end of the sort string
                        // This may result in redundancy... but it seems to work
                        $sortterm .= ($sortterm ? ', ' : '') . $defaultSort;
```